packages/cli/commands/projects.js

```diff
@@ -0,0 +1,19 @@
+const {
+  addConfigOptions,
+  addAccountOptions,
+  addOverwriteOptions,
+} = require('../lib/commonOpts');
+const deploy = require('./projects/deploy');
+
+exports.command = 'projects';
+exports.describe = false; //'Commands for working with projects';
```

Comment:
I'm new to yargs. Is this what's causing the projects command to be hidden?

Reply:
Yes

Reply:
Yes, if you return `false`, nothing will show in the `--help` section for the command.